docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -138,7 +367,54 @@ TBD
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use atparse-formatted template file ``atparse.txt`` with content
+
+.. code:: sh
+  @[greeting], @[recipient]!
```

Comment:
For some reason, this block is not showing up for me in the preview HTML docs.

Reply:
Ah, I thought I caught all the missing `.. code::` line breaks, thanks!